cg/test/management/users.api.spec.js

```diff
@@ -154,3 +154,70 @@ describe('User password update', () => {
     expect((await res2.json()).jwt).toBeTruthy();
   });
 });
+
+describe('Listing management users', () => {
+  it('Should display an empty list when no manager is registered', async () => {
+    //Given
+    await db('users').del(); // It's fine to delete the contents of the managers table because no other table references it
+
+    //When
+    const res = await fetch('/api/management/users/list', {
+      method: 'GET',
+      headers: {
+        Authorization: `Bearer ${managementToken}`
+      }
+    });
+
+    //Then
+    expect(res.ok).toBeTruthy();
+    expect(res.status).toBe(200);
+    expect(await res.json()).toEqual(expect.arrayContaining([]));
```

Comment:
This is wrong. It will pass even if the array isn't empty.
It should be:
`expect(await res.json()).toEqual([]);`

see this: https://repl.it/@maksymilian_maj/jest-arrays